tests/manage/pv_services/test_del_mon_service_and_create_pvc.py

```diff
@@ -0,0 +1,212 @@
+            mon_endpoint = f"{cluster_ip}:{port}"
+            mon_id = svc["spec"]["selector"]["mon"]
+            list_old_svc.append(cluster_ip)
+
+            # Delete deployment
+            log.info("Delete mon deployments")
+            del_obj = OCP(
+                kind=constants.DEPLOYMENT,
+                namespace=constants.OPENSHIFT_STORAGE_NAMESPACE,
+            )
+            del_obj.delete(resource_name=name)
+
+            # Delete pvc
+            log.info("Delete mon PVC")
+            pvc_obj = OCP(
+                kind=constants.PVC, namespace=constants.OPENSHIFT_STORAGE_NAMESPACE
+            )
+            pvc_obj.delete(resource_name=name)
+
+            # Delete the mon service
+            log.info("Delete mon service")
+            svc_obj = OCP(
+                kind=constants.SERVICE, namespace=constants.OPENSHIFT_STORAGE_NAMESPACE
+            )
+            svc_obj.delete(resource_name=name)
+
+            # Edit the cm
+            log.info(f"Edit the configmap {constants.ROOK_CEPH_MON_ENDPOINTS}")
+            configmap_obj = OCP(
+                kind=constants.CONFIGMAP,
+                namespace=constants.OPENSHIFT_STORAGE_NAMESPACE,
+            )
+            output_get = configmap_obj.get(
+                resource_name=constants.ROOK_CEPH_MON_ENDPOINTS
+            )
+            new_data = output_get["data"]
+            new_data["csi-cluster-config-json"] = (
+                new_data["csi-cluster-config-json"].replace(f'"{mon_endpoint}",', "")
+                if new_data["csi-cluster-config-json"].find(f'"{mon_endpoint}",') != 1
+                else new_data["csi-cluster-config-json"].replace(
+                    f',"{mon_endpoint}"', ""
+                )
+            )
+            new_data["data"] = ",".join(
+                [
+                    value
+                    for value in new_data["data"].split(",")
+                    if f"{mon_id}=" not in value
+                ]
+            )
+            new_data["mapping"] = (
+                new_data["mapping"].replace(f'"{mon_id}":null,', "")
+                if new_data["mapping"].find(f'"{mon_id}":null,') != -1
+                else new_data["mapping"].replace(f',"{mon_id}":null', "")
+            )
+            params = f'{{"data": {json.dumps(new_data)}}}'
+            log.info(f"Removing {mon_id} entries from configmap")
+            configmap_obj.patch(
+                resource_name=constants.ROOK_CEPH_MON_ENDPOINTS,
+                params=params,
+                format_type="strategic",
+            )
+            log.info(
+                f"Configmap {constants.ROOK_CEPH_MON_ENDPOINTS} edited successfully"
+            )
+
+            # Scale up rook-ceph-operator
+            log.info("Scale up rook-ceph-operator")
+            assert modify_deployment_replica_count(
+                deployment_name="rook-ceph-operator", replica_count=1
+            ), "Failed to scale up rook-ceph-operator to 0"
```

Comment:
It should be , Failed to scale up rook-ceph-operator to 1

Reply:
oops, thanks for pointing it out